src/content/en/android/trusted-web-activity/whats-new/index.md

```diff
@@ -0,0 +1,461 @@
+project_path: /web/android/_project.yaml
+book_path: /web/android/_book.yaml
+description: Learn about the new features for web applications that use Trusted Web Activity.
+
+{# wf_published_on: 2020-11-25 #}
+{# wf_updated_on: 2020-11-25 #}
+{# wf_tags: trusted-web-activity #}
+{# wf_featured_image: /web/updates/images/generic/devices.png #}
+{# wf_blink_components: N/A #}
+
+# What’s new for Web In Play {: .page-title }
+
+{% include "web/_shared/contributors/andreban.html" %}
+{% include "web/_shared/contributors/adrianajara.html" %}
+
+Since [Trusted Web Activity][1] was introduced last year, the Chrome team continues to work on the
+product, making it easier to use with Bubblewrap, adding new features like the upcoming Google Play
+Billing integration, and enabling it to work on more platforms, like Chrome OS. This article will
+summarize the latest and upcoming updates for Trusted Web Activity.
+
+## New Bubblewrap and Trusted Web Activity features
+
+[Bubblewrap][2] helps you create apps that launch your PWAs inside a Trusted Web Activity, without
+requiring knowledge of platform-specific tooling.
+
+### Simplified setup flow
+
+Previously, using Bubblewrap required manually setting up the Java Development Kit and the Android
+SDK, both of which are error prone. The tool now offers to automatically download the external
+dependencies when run for the first time.
+
+You can still choose to use an existing installation of the dependencies, if you prefer to do so,
+and the new `doctor` command helps find issues and recommends fixes to the configuration, which can
+now be updated from the command line using the `updateConfig` command.
+
+### Improved Wizard
+
+When creating a project with `init`, Bubblewrap needs information to generate the Android app. The
+tool extracts values from the Web Manifest and provides defaults where possible.
```

Comment:
Web App Manfest?

Reply:
I should have caught this before. Is this the Web App Manifest or something particular to TWAs?